docs/SUMMARY.md

```diff
@@ -37,6 +37,7 @@
   * [Authentication](recipes/authentication.md)
   * [Caching](recipes/cache.md)
   * [Debouncing Actions](recipes/debouncing-actions.md)
+  * [Actions as DOM Events](recipes/actions-as-dom-events.md)
```

Comment:
These seem to be alphabetical

Reply:
`Actions As DOM Events`? or prolly I'm missing something... :smile:

Reply:
The menu items seem to be alphabetical and this item is then out of order.

Reply:
Gotcha